Libraries/Lists/SectionList.js

```diff
@@ -317,6 +317,13 @@ class SectionList<SectionT: SectionBase<any>> extends React.PureComponent<
     }
   }
 
+  setNativeProps(props: Object) {
+    const listRef = this._wrapperListRef && this._wrapperListRef.getListRef();
```

Comment:
It will not work if `legacyImplementation` prop is true, because `VirtualizedSectionList` doesn't have `setNativeProps`.